kubejs/server_scripts/railways/tags.js

```diff
@@ -7,8 +7,12 @@ const registerRailWaysItemTags = (event) => {
         event.removeAllTagsFrom(item)
         event.add('c:hidden_from_recipe_viewers', item)
     })
+    // Adding Conductor tags
+    global.MINECRAFT_DYE_NAMES.forEach(dye => {
+		if (dye != 'white') event.add('tfg:colored_caps', `railways:${dye}_conductor_cap`)
+		event.remove('c:hidden_from_recipe_viewers',`railways:${dye}_conductor_cap`)
```

Comment:
you can stop the tag from being added in the first place in startup_scripts/railways/constants.js